addons/interact_menu/functions/fnc_collectActiveActionTree.sqf

```diff
@@ -43,12 +43,14 @@ _origActionData params [
     "_distance"
 ];
 
+private _result = [_target, ACE_player, _customParams] call _conditionCode;
+
 // Return nothing if the action itself is not active
-if !([_target, ACE_player, _customParams] call _conditionCode) exitWith {
+if (isNil "_result" || {!_result}) exitWith {
```

Comment:
```suggestion
if (isNil "_result" || {!(_result isEqualType true)} || {!_result}) exitWith {
```
can do safer but no script error when bad value (not tested)

Reply:
I know, but decided against it because of no script errors. `nil` just doesn't handle anything properly, that's why are handling it specially.